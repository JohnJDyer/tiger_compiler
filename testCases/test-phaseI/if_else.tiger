let
in
  if 1+2 then
    flush();
  else
    flush();
  endif;
end