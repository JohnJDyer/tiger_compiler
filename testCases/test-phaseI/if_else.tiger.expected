main:
	add, 1, 2, $t0
	breq, $t0, 0, if_label0
	call, flush
	goto, if_label1, , 
if_label0:
	call, flush
if_label1:
	return, , , 
